.markdown-link-check.jsonc

```diff
@@ -23,5 +23,12 @@
         "replacement": "{{BASEURL}}/"
       }
     ],
-    "aliveStatusCodes": [200, 206]
+    "aliveStatusCodes": [200, 206],
```

Comment:
@roel4ez can we make these changes a separate pr?

thanks

Reply:
done, see #1427 - which would need to be merged before this.